core/src/main/java/com/opentext/ia/sdk/support/io/DomainObjectTooBigException.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2016-2017 by OpenText Corporation. All Rights Reserved.
+ */
+package com.opentext.ia.sdk.support.io;
+
+/**
+ * Extends {@linkplain RuntimeException}. Used when a DomainObject is too big to fit in a SIP with the defined limit.
+ */
+public class DomainObjectTooBigException extends RuntimeException {
+
+  private static final long serialVersionUID = -3456477572689104546L;
+
+  private final long domainObjectSize;
+  private final long maxSipSize;
+
+  public DomainObjectTooBigException(long inDomainObjectSize, long inMaxSipSize) {
```

Comment:
Follow the convention for assigning fields in a constructor, e.g. `this.maxSipSize = maxSipSize`.